src/Compatibility/Core/src/iOS/Renderers/NavigationRenderer.cs

```
@@ -764,6 +764,11 @@ void UpdateBarTextColor()
 
 		void SetStatusBarStyle()
 		{
+			if (NavPage is null)
```

Although the fix looks correct, this changes worry me that we could be hiding an error or the real reason, do we know the origin of the error? Maybe we should check `HasNavigationBar` and avoid invalidating something first.

---

Could be really nice if we can achieve to have a test, because in that way we could know exactly what's happening. Reading the issue seems to be related with use a FlyoutPage and HasNavigationBar, but not sure how to reproduce it.

---

Yea, but, I’m currently unable to reproduce the issue. I’m waiting for the reporter to either provide clear reproduction steps or confirm whether this PR resolves the crash.